src/test/run-make/issue-36710/Makefile

```diff
@@ -1,6 +1,10 @@
-# ignore-cross-compile $(call RUN,foo) expects to run the target executable natively
+# ignore-riscv64 $(call RUN,foo) expects to run the target executable natively
 #                              so it won't work with remote-test-server
+# ignore-arm Another build using remote-test-server
 # ignore-none no-std is not supported
+# ignore-wasm32 FIXME: don't attempt to compile C++ to WASM
+# ignore-wasm64 FIXME: don't attempt to compile C++ to WASM
+# ignore-nvptx64-nvidia-cuda FIXME: can't find crate for `std`
```

Comment:
Why is this a FIXME? How would CUDA ever get support for libstd? (Is there some way we can say "ignore all targets without std"?)

Reply:
The commit message adding this originally says https://github.com/rust-lang/rust/commit/1fa48cf181f1fe7e1aba133be199804c652fe55c
> nvtptx64-nvidia-cuda fails in rustc saying it can't find std. The rust
platforms support page says that std is supported on cuda so this is
surprising.

sounds like *something* is broken here but no need to fix it in your PR. cc @RDambrosio016 in case you know who maintains this target.